tasks/install.yml

```
@@ -54,23 +54,23 @@
 
 - name: Check promtail binary
   stat:
-    path: "{{ promtail_install_dir }}/{{ promtail_version }}/promtail-linux-amd64"
+    path: "{{ promtail_install_dir }}/{{ promtail_version }}/promtail-linux-{{ go_arch }}"
```

would it make sense to shorten the path to 
```suggestion
    path: "{{ promtail_install_dir }}/{{ promtail_version }}/promtail"
```